app/assets/stylesheets/application.scss

```diff
@@ -17,6 +17,13 @@
  *= require owl.theme
 
  */
+
+ $primary-color: #09B1D8;
```

Comment:
Color `#09B1D8` should be written as `#09b1d8`<br>Line should be indented 0 spaces, but was indented 1 spaces